src/test/config_test.ts

```diff
@@ -219,6 +221,25 @@ suite('config', () => {
       assert.deepEqual(actual, expected);
     });
 
+    test(
+        'it updates an on-disk config file with the schema, if absent',
```

Comment:
remove "it"

Reply:
Dropped the test, as the code is now better organized, but a bit harder to test.

Reply:
Yeah, I need to add an end-to-end test, and once we have that rig set up we could add it back properly.